frontend/src/metabase/static-viz/components/XYChart/XYChart.tsx

```diff
@@ -267,7 +270,7 @@ export const XYChart = ({
             <GoalLine
               label={settings.goal.label}
               x1={0}
-              x2={innerWidth}
+              x2={calculatedInnerWidth}
```

Comment:
Fix grid lines are not the same size as the axis

#### After
![image](https://user-images.githubusercontent.com/1937582/188827991-73761958-995f-42fb-820d-1b573f3ee37c.png)

#### Before
![Screen Shot 2022-09-07 at 3 19 03 PM](https://user-images.githubusercontent.com/1937582/188828010-6832bc89-94d9-4a04-9abd-237c4c27bc38.png)

It's a bit hard to see, but the grid lines extend beyond the x-axis